libopae/plugins/xfpga/sysfs.c

```diff
@@ -736,13 +758,26 @@ fpga_result sysfs_get_fme_temp_path(fpga_token token, char *sysfs_temp)
 	}
 
 	res = cat_token_sysfs_path(sysfs_temp, token, SYSFS_FORMAT(sysfs_fme_temp_glob));
+	if (res != FPGA_OK) {
+		return res;
+	}
+
+	// check for path is valid
+	int gres = glob(sysfs_temp, GLOB_NOSORT, NULL, &pglob);
```

Comment:
We should probably if glob returns GLOB_NOSPACE and translate that into an FPGA_NO_MEMORY result
Maybe translate GLOB_NOMATCH to FPGA_NOT_FOUND and GLOB_ABORTED into FPGA_EXCEPTION?

Reply:
Fiexd